en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md

```diff
@@ -1,6 +1,8 @@
 # BigQuery Connector Example
 
-The BigQuery connector allows you to access the [BigQuery REST API](https://cloud.google.com/bigquery/docs/reference/rest) through WSO2 EI. BigQuery is a tool that allows you to execute SQL-like queries on large amounts of data at outstanding speeds. It is a serverless Software as a Service that supports querying using ANSI SQL.
+The BigQuery connector allows you to access the [BigQuery REST API](https://cloud.google.com/bigquery/docs/reference/rest) through WSO2 EI. BigQuery is a tool that allows you to execute SQL-like queries on large amounts of data at outstanding speeds. It is a serverless Software as a Service that supports querying using ANSI SQL. The WSO2 EI BigQuery connector uses user credentials to ensure that the application has access only to BigQuery tables that are available to the end user.
```

Comment:
The WSO2 EI BigQuery connector uses user credentials to ensure that the application has access only to BigQuery tables that are available to the end user.

- it is not clear what is meant by this? Could you explain more?

Reply:
Removed this section.